src/flamegpu/gpu/CUDASimulation.cu

```diff
@@ -1234,6 +1236,29 @@ void CUDASimulation::simulate() {
     }
     processExitLog();
 
+    // Send Telemetry
+    if (getSimulationConfig().telemetry) {
+        // Generate some payload items
+        std::map<std::string, std::string> payload_items;
+        payload_items["GPUDevices"] = flamegpu::util::detail::compute_capability::getDeviceName(deviceInitialised);
+        payload_items["SimTime(s)"] = std::to_string(elapsedSecondsSimulation);
+        // generate telemtry data
+        std::string telemetry_data = flamegpu::io::Telemetry::generateTelemetryData("simulation-run", payload_items);
+        // send
+        if (!flamegpu::io::Telemetry::sendTelemetryData(telemetry_data)) {
+            if ((getSimulationConfig().verbosity > Verbosity::Quiet))
+                fprintf(stderr, "Warning: Usage statistics for CUDASimulation failed to send.\n");
```

Comment:
Might want to only show this in verbose, not unreasonable that FLAMEGPU might get ran on systems without access to the telemetry server. (It's not a warning that a user should care about)